<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BDWindow</class>
 <widget class="QDialog" name="BDWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="chrNameListWidget"/>
   </item>
   <item row="0" column="1">
    <widget class="QTreeWidget" name="bdTreeWidget">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="showBDButton">
     <property name="text">
      <string>showBD</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
